frontend/src/scenes/billing/UnsubscribeSurveyModal.tsx

```diff
@@ -68,7 +89,7 @@ export const UnsubscribeSurveyModal = ({
                 ) : (
                     <LemonBanner type="info">
                         <p>
-                            Your invoice will be billed immediately.{' '}
+                            Any outstanding invoices will be billed immediately.{' '}
```

Comment:
Note: need to figure out what happens for outstanding invoices when the customer is removing an addon.